Samples/auth/Outlook-Add-in-SSO-NAA-Identity/README.md

```diff
@@ -0,0 +1,276 @@
+products:
+  - office
+  - office-outlook
+languages:
+  - javascript
+extensions:
+  contentType: samples
+  technologies:
+    - Add-ins
+  createdDate: "01/14/2025 10:00:00 AM"
+description: "This sample shows how to send identity claims across the network to resources like a web API service."
+---
+
+# Send identity claims to resources using nested app authentication (NAA) and SSO
+
+The scenario for this sample is that you need to send the signed-in user's identity claims (such as name, email, or a unique ID) to a resource such as a database. This enables you to associate and track data for each user in the resource. This sample replaces an obsolete pattern for legacy Exchange Online tokens. The obsolete pattern is that Exchange Online identity tokens were passed over the network to share user identity. Using Entra ID tokens, the ID token must never be passed by your Outlook add-in over the network to any resource. It can only be used within the context of the add-in (such as the task pane) itself. To pass user identity over the network, you use the Entra ID access token which contains the identity claims.
+
+This sample shows how to implement a todo list for multiple users with an Outlook add-in.
+
+1. The Outlook add-in (SPA client) uses the MSAL.js library to obtain an access token from Microsoft Entra ID. The MSAL.js library returns an ID token, access token, and refresh token.
+1. The access token is passed in a web request to a protected web API service. The web request validates the access token and confirms the user has permissions to perform the action in the API. The access token contains any identity claims the web API needs.
+
+![A diagram showing the Outlook add-in getting an access token from Microsoft Entra ID and calling the protected web API with the access token.](architecture-diagram.png)
+
+## Applies to
+
+- Outlook on Windows (new and classic), Mac, mobile, and on the web.
+
+## Prerequisites
+
+- Office connected to a Microsoft 365 subscription (including Office on the web).
+- [Node.js](https://nodejs.org/) version 16 or greater.
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 8 or greater.
+
+## Register the sample applications with your Microsoft Entra tenant
+
+### Register the web API service
+
+The web API service is a Node.js server that requires an application registration to authorize calls to the todo list database.
+
+1. To register your app, go to the [Azure portal - App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) page.
+1. Sign in with the **_admin_** credentials to your Microsoft 365 tenancy. For example, **MyName@contoso.onmicrosoft.com**.
+1. Select **New registration**. On the **Register an application** page, set the values as follows.
+
+   - Set **Name** to `Contoso-Web-API-Server`.
+   - Set **Supported account types** to **Accounts in this organizational directory only**.
+   - Select **Register**.
+
```

Comment:
The second time we register an application here, we are told to note the Application (client) ID. For this first application, we are told to grab the ID from the Microsoft Entra admin center when needed. I'd recommend avoiding sending people to the other admin center (unless that's what I'm messing up).

Reply:
I removed mentions of the Entra admin center so it all focused on the portal now.

Reply:
I removed all references to Microsoft Entra admin center.